client/me/purchases/manage-purchase/index.jsx

```diff
@@ -254,98 +269,145 @@ const ManagePurchase = React.createClass( {
 				{ translate( 'Cancel Privacy Protection' ) }
 			</CompactCard>
 		);
-	},
+	}
 
-	renderPurchaseDetail() {
```

Comment:
With these changes we might be losing substantial information about some purchase types. For example, if you purchased a theme, in the detail page you might not see that it's actually a theme:

![](https://cldup.com/UjfHwlbl2X.png)

while previously it was indicating that it was a premium theme and it included a link to the theme page:

![](https://cldup.com/ZgiMl_LrJr.png)

We should make sure to check that with all purchase types.